src/pages/EditMonitor.vue

```diff
@@ -434,6 +437,19 @@
                                 </div>
                             </template>
 
+                            <!-- Zookeeper -->
+                            <template v-if="monitor.type === 'zookeeper'">
+                                <div class="my-3">
+                                    <label for="zookeeperHost" class="form-label">{{ $t("Zookeeper Host") }}</label>
+                                    <input id="zookeeperHost" v-model="monitor.zookeeperHost" type="text" class="form-control" required>
```

Comment:
Please include a helptext about what you mentioned in the PR-description.
Currently, a user might just think it is not working, right?